server/db/db.js

```diff
@@ -797,6 +797,46 @@ async function createClient(displayName, fromPhoneNumber, responderPhoneNumber,
   }
 }
 
+async function getNewNotificationsCountByAlertApiKey(alertApiKey, clientParam) {
+  try {
+    const { rows } = await runQuery(
+      'getNewNotificationsCountByAlertApiKey',
+      'SELECT COUNT (*) FROM notifications n LEFT JOIN locations l ON n.location_id = l.locationid LEFT JOIN clients c ON l.client_id = c.id WHERE c.alert_api_key = $1 AND NOT n.is_acknowledged',
```

Comment:
So this could change to not need the join to the `locations` table.

And while changing this, it would be good to use `helpers.runQuery` instead of just `runQuery` and it's easier to read if you use the backquotes and more newlines in the query string.